platform/admin/docs/src/main/resources/ManContents.adoc

```diff
@@ -237,3 +237,28 @@ A system restart is required for the changes to `users.properties` to take effec
 *Web Context Policy Manager*
 
 image::web_context_policy_manager.png[Web Context Policy Manager,${image.width}]
+
+=== Exporting Configurations
+
+You can export the current system configurations using the Admin UI.  This is useful for migrating from one running instance to another.
+
+To do so, follow these instructions:
+
+. Select the `System` tab (next to the Applications tab) image:exporting_configuration_step1.png[Exporting Step 1,${image.width}]
+. Click the `Export` button image:exporting_configuration_step2.png[Exporting Step 2,${image.width}]
+. Fill out the form, specifying where you want to export the configurations image:exporting_configuration_step3.png[Exporting Step 3,${image.width}]
```

Comment:
Do we want to mention that the path points to a directory on the server, not on the user's local machine?